mkdocs/docs/getting-started/install.md

```diff
@@ -10,13 +10,14 @@ hide:
 Install Iter8 in your Kubernetes cluster as follows.
 
 ```shell
-export TAG=v0.5.2
-kubectl apply -f https://raw.githubusercontent.com/iter8-tools/iter8-install/$TAG/core/build.yaml
+export TAG=master
+kustomize build github.com/iter8-tools/iter8/install/core/?ref=${TAG} | kubectl apply -f -
+kubectl wait --for=condition=Ready pods --all -n iter8-system
 ```
 
-The above command installs Iter8's controller and analytics services in the `iter8-system` namespace, the Experiment and Metric CRDs, and the following RBAC permissions.
+To pin the version of Iter8, replace `master` with `v0.5.13` as the exported TAG. The above command installs Iter8's controller and analytics services in the `iter8-system` namespace, Iter8's experiment and metric CRDs, and the following RBAC rules.
```

Comment:
Should we say "To pin the version of Iter8, replace `master` with the appropriate tag number (for example, `v0.5.13`) as the exported `TAG`"?

Reply:
@Alan-Cha For v0.5, the only appropriate tag number happens to be `v0.5.13`. From v0.6 onwards, what you mention above is how we will document it. This is because, we just recently changed the install process...